text/0000-guaranteed-tco.md

```diff
@@ -0,0 +1,602 @@
+- Feature Name: guaranteed_tco
+- Start Date: 2023-04-01
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
```

Comment:
I think this usually done upon acceptance.

Reply:
yeah, it's left at `0000` at least to make it show up first on the list while it's being reviewed

Reply:
Well it would be the first RFC where this step is forgotten. :slightly_smiling_face:
There is also a set procedure to finish up RFCs, I expect this change will be done once it gets to that point.